process/support_team.md

```diff
@@ -4,43 +4,47 @@
 * Fix issue if it can be done so in < 2 hours
 * Fix issue if it is causing significant business impact to customer
-* If it is a larger piece of work, and/or non-urgent then it can be moved to the customer's backlog in Trello.
+* If it is a larger piece of work, and/or non-urgent then it should be moved to the customer's backlog in Trello, where it can be prioritised against other work, and potentially scheduled in for a Support Pair to complete.
 
 Each Customer has their own Trello board for prioritising work. Issues are grouped differently for each customer but each board has the following 3 columns:
 1. 'New Issues' this is anything that's been backlogged from a Groove issue.
-2. 'Up next' issues will be automatically copied to our Support Epics board
+2. 'Up next' issues will be automatically copied to our Support Epics board where they can be scheduled and assigned.
 3. 'Recently Completed' issues are things which have been completed in the last week or so.
 
-During a weekly (bi-weekly for some customers) call with each customer Nick and the custmer will agree to a tentative commitment of work to be completed the following week by the support team. These issues will be moved to the 'Up Next' column, and copied to the Support Epics board.
-
-Every Monday everyone working on support that week will have a planning session to divide up the work for our customers across the team. Following this session Nick will follow up with each costomer to confirm what's going to be done that week.
-
-Every day support team members should attend the daily standup to report their progress ontheir support assignments, and flag up any issues or impediments which may impact the team's commitment that week.
-
-Once an issue is complete, the customer should be notified and the issue moved to the 'Done' column for the current week. 
-
-## Working in Support
-
-If you're working on support in a week (or part of a week) then you should:
- * Attend the planning session at the start of the week, please come armed with the number of days you can commit to support that week.
- * Attend stand-ups on days you are doing support work.
- * Where possible we like to pair on issues, especially for our more esoteric projects, in order to spread domain knowledge throughout the company, so come willing and able to do so.
+## Support Planning
 
-### The Triage Team
+During a weekly call (bi-weekly for some) with each customer Nick and the customer will agree to a commitment of work to be completed the following week by the support team. These issues will be moved to the 'Up Next' column, and copied to the Support Epics board.
 
-As part of your support week you might want to volunteer to be on the triage team. This generally consists of one of Nick, Luke, Chris, Rory plus one other person.
+By Thursday of each week the Support Epics board will have:
```

Comment:
What is the definition of "reasonable". A top-down approach or a bottom-up approach?